amd_openvx_extensions/amd_rpp/include/vx_ext_rpp.h

```diff
@@ -982,10 +982,10 @@ extern "C"
 	/*! \brief [Graph] Creates a RPP Brightness function node.
 	 * \ingroup group_amd_rpp
 	 * \param [in] graph The handle to the graph.
-	 * \param [in] pSrc The input tensor in <tt>\ref VX_TYPE_UINT8<tt> or <tt>\ref VX_TYPE_FLOAT32<tt> or 
```

Comment:
Please revert these changes . This need not be a part of the audio augmentations pr

Reply:
Done